### DISCUSSION TITLE
How to define the return button

### DISCUSSION THREAD

**Discussion by tsiens** [flet-dev/flet#3788] [Q&A]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the bug

I have a keypad phone，I need to tamper with the system buttons。
When I press the back button, I will exit the app and return to the desktop。
May I ask how to disable the return button and customize it

### Code sample

in python-for-android，i can modify PythonActivity.java
```
    @Override
    public void onBackPressed() {
        mWebView.evaluateJavascript("javascript:back()",new ValueCallback<String>(){	    
            @Override
            public void onReceiveValue(String s) {}
        });
        return;
    }

    @Override
    public boolean onKeyDown(int keyCode, KeyEvent event) {
        if (keyCode == KeyEvent.KEYCODE_DPAD_UP | keyCode == KeyEvent.KEYCODE_DPAD_DOWN | keyCode == KeyEvent.KEYCODE_DPAD_LEFT | keyCode == KeyEvent.KEYCODE_DPAD_RIGHT) {
            return true;
        }
        return super.onKeyDown(keyCode, event);
    }
```

### To reproduce

-

### Operating System

Linux

### Operating system details

android 11

### Flet version

0.23.2

### Regression

No, it isn't

**Comment by Laztef** [comment]
any solution for this?
